handler/main.go

```diff
@@ -2,13 +2,66 @@ package main
 
 import (
 	"fmt"
+	"os"
```

Comment:
Why didn't you process the `sheets` environment variable with the rest of the environment variables using `/caarlos0/env/` in `inv/inv.go`?